kebab/src/raw.ts

```diff
@@ -80,9 +81,8 @@ export const connect = (
       WebSocket,
     });
     const apiSend = (opcode: Opcode, data: unknown, fetchId?: FetchID) => {
-      const raw = `{"op":"${opcode}","d":${JSON.stringify(data)}${
-        fetchId ? `,"fetchId":"${fetchId}"` : ""
-      }}`;
+      const raw = `{"op":"${opcode}","d":${JSON.stringify(data)}${fetchId ? `,"fetchId":"${fetchId}"` : ""
+        }}`;
```

Comment:
fix your auto formatter

Reply:
what's wrong here?

Reply:
your vscode isn't using prettierjs

Reply:
oh i see, wtf

Reply:
> your vscode isn't using prettierjs

wait, i did install that extension ben told